website/server/models/newsPost.js

```diff
@@ -0,0 +1,73 @@
+import mongoose from 'mongoose';
+import baseModel from '../libs/baseModel';
+
+const { Schema } = mongoose;
+
+export const schema = new Schema({
+  title: { $type: String },
+  author: { $type: String },
+  credits: { $type: String },
+  publishDate: { $type: Date },
+  published: { $type: Boolean },
+  text: { $type: String },
+}, {
+  strict: true,
+  minimize: false, // So empty objects are returned
+  typeKey: '$type', // So that we can use fields named `type`
+});
+
+schema.plugin(baseModel, {
+  noSet: ['_id'],
+  timestamps: true,
+});
+
+schema.statics.getNews = async function getNews (isAdmin) {
+  let posts = [];
+  if (!isAdmin) {
+    posts = this.find({
+      published: true,
+      publishDate: { $lte: Date() },
```

Comment:
is this necessary? I mean we're filtering for things published and not published in the future?

If we keep it, it should be `new Date()`

Reply:
This allows us to create posts in advance, that automatically go live on the day. Probably not 100% necessary, but imho a nice feature.